docs/environment.md

```diff
@@ -0,0 +1,108 @@
+# Environment Variables
+
+This guide provides a list of environment variables that can be set that impact the execution of Cog. This assumes that you have already followed the [main steps to set up Cog](https://github.com/replicate/cog/blob/main/docs/getting-started.md) and have a general familiarity with how to run Cog.
+
+
+## General
+
+This section lists the relevant environment variables for general execution of Cog.
+
+### CGO_ENABLED
```

Comment:
```suggestion
### `CGO_ENABLED`
```

Let's put these all in backticks to make it clear that they're "code".

Reply:
Updated!